web/src/features/fireWeather/components/graphs/PrecipGraph.tsx

```diff
@@ -267,17 +364,50 @@ const PrecipGraph: React.FunctionComponent<Props> = ({
       .add(6, 'hours')
       .toDate()
     const xDomain: [Date, Date] = [d1, d2]
-    maxPrecip = Math.ceil(maxPrecip / 10) * 10 // round to the nearest 10
+
+    // Determine maxDailyPrecip for yScale of graph
+    const dailyPrecipVals: number[] = []
```

Comment:
You can actually create this at the beginning and push elements when processing them. Then pull max value at the end using `d3.max`. That's how it's done to get the max and min of temp in TempRHGraph. Can you modify this part so that it matches?